### PR TITLE
Optional format

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: src/formatter.rs [hunk 1]
```diff
@@ -77,8 +79,10 @@ impl<'a> Formatter for RedisJsonFormatter<'a> {
         self.current_indent -= 1;
 
         if self.has_value {
-            writer.write_all(self.newline)?;
-            Self::indent(writer, self.current_indent, self.indent)?;
+            if let Some(n) = self.newline.as_ref() {
+                writer.write_all(n.as_bytes())?;
```
Comment (oshadmi) [inline src/formatter.rs:83]: Since repeated 4 times, can be considered to be moved to `RedisJsonFormatter` impl (to write an optional string)

**Review by oshadmi — APPROVED** [review]
👍🏼

**Review by gavrie — CHANGES_REQUESTED** [review]
LGTM, just @oshadmi 's comment.

**Review by oshadmi — APPROVED** [review]

**Review by gavrie — APPROVED** [review]

File: src/formatter.rs (hunk 1 above)
Comment (gkorland) [reply src/formatter.rs]: @oshadmi please check the last change

**Review by oshadmi — APPROVED** [review]
👍🏼

**Review by gavrie — APPROVED** [review]

**Pull request merged**